.github/workflows/registry.yml

```diff
@@ -93,14 +85,29 @@ jobs:
         run: |
           rsync -r $LEGACY_HELM_CHART $HELM_CHARTS_DIR
           rsync -r $HELM_CHART/* $LEGACY_HELM_CHART --exclude=Chart.yaml --exclude=crds
+      - uses: oprypin/find-latest-tag@v1
+        with:
+          repository: oam-dev/kubevela
+          releases-only: true
+        id: latest_tag
       - name: Tag helm chart image
         run:  |
-          version=${{ steps.get_version.outputs.VERSION }}
-          sed -i "s/latest/$version/g" $HELM_CHART/values.yaml
-          sed -i "s/latest/$version/g" $LEGACY_HELM_CHART/values.yaml
-          number=${version#"v"}
-          sed -i "s/0.1.0/$number/g" $HELM_CHART/Chart.yaml
-          sed -i "s/0.1.0/$number/g" $LEGACY_HELM_CHART/Chart.yaml
+          latest_tag=${{ steps.latest_tag.outputs.tag }}
+          sub="."
+          major="$(cut -d"$sub" -f1 <<<"$latest_tag")"
+          minor="$(cut -d"$sub" -f2 <<<"$latest_tag")"
```

Comment:
what will happen if the cut failed, for example the latest tag become v1.0 for some reason.

Reply:
All tags should follow the rule of `major.minor.patch` or `major.minor` if we wanna to build a valid helm chart. If the `cut` failed, it's